app/models/person.rb

```diff
@@ -7,4 +7,8 @@ class Person < ApplicationRecord
   def home_address?
     home_address.address_line_one.present?
   end
+
+  def full_name
```

Comment:
Yep, probably a decorator a bit down the line, along with some other bits that will surely be required for the dashboard table, but for now is perfectly ok.